src/electron/electron.html

```diff
@@ -2,12 +2,10 @@
 
 <body>
 	<script>
-		const { ipcRenderer } = require('electron');
+	const { ipcRenderer } = require('electron');
 
-		ipcRenderer.on('message', function (event, text) {
-			console.log('auto-updater message:', text);
-		})
-	</script>
+ipcRenderer.on('message', function(event, text) {});
```

Comment:
When we don't have a function that does anything this can be removed :)

Reply:
I wonder if line 5 should be removed too... `const { ipcRenderer } = require('electron');`